docs/cep/query-guide/functions/unique/index.md

```diff
@@ -0,0 +1,36 @@
+|------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [deduplicate (Stream Processor)](unique/deduplicate)          | Duplicate events are removed based on the `unique.key` arriving in `time.interval`.                                                                                          |
+| [ever (Window)](unique/ever)                                  | Retains latest events based on the `unique keys`.                                                                                                                            |
+| [externalTimeBatch (Window)](unique/externalTimeBatch)         | Tumbling time window holds the latest events based on `unique.key` and various parameters for time.                                                                          |
+| [first (Window)](unique/first)                  | Retain first set of unique events based on `unique.key`.                                                                                                                     |
+| [firstLengthBatch (Window)](unique/firstLengthBatch)             | Tumbling window holds `window.length` unique events based on `unique.key`                                                                                                    |
+| [firstTimeBatch (Window)](unique/firstTimeBatch)               | A batch-time or tumbling window that holds the unique events based on `unique.key` and sliding time period `window.time` with time offset defined by parameter `start.time`	 |
+| [length (Window)](unique/length)                    | A sliding length window that holds the latest unique events base in `unique.key`.                                                                                            |
+| [lengthBatch (Window)](unique/lengthBatch)              | This is a batch (tumbling) window that holds `window.length` number of latest unique events.                                                                                 |
+| [time (Window)](unique/time)                      | This is a sliding time window that holds the latest unique events that arrived during `window.time` .                                                                        |
+| [timeBatch (Window)](unique/timeBatch)                 | This is a batch (tumbling) time window that is updated with the latest events based on `unique.key` tumbling time period `window.time` and time offset `start.time`.         |
+| [timeLengthBatch (Window)](unique/timeLengthBatch) | The window tumbles upon the elapse of the time window `window.time`, or when a number of unique events have arrived i.e. `window.length`.                                    |
+
+## Example
```

Comment:
Please provide some context and explanation for the example. Pretend you are explaining it to me, a person who has never seen it before and is fuzzy on what windows and stream workers are. Use simple, plain language.